client/core/core.go

```diff
@@ -2875,6 +2877,46 @@ func (c *Core) tipChange(assetID uint32, nodeErr error) {
 	c.updateBalances(counts)
 }
 
+func (c *Core) ShutdownWithActiveOrders() bool {
+	c.connMtx.Lock()
+	defer c.connMtx.Unlock()
+	ok := true
+	for _, dc := range c.conns {
+		if dc.hasActiveOrders() {
+			ok = false
+			break
+		}
+	}
+	if !ok {
+		fmt.Print("Can not shutdown with active orders, do you want to shutdown anyway? (y or enter to abort): ")
```

Comment:
How about "You have active orders. Shutting down now may result in failed swaps and account penalization. Do you want to quit anyway? ('y' to quit, enter to abort):"